### PR TITLE
finish login

### PR DESCRIPTION
**Pull request by mohamed-ahmed-yousef**

### CODE REVIEW

File: lib/axios.ts [hunk 1]
```diff
@@ -6,8 +6,9 @@ export const http = axios.create({
 });
 
 http.interceptors.request.use(async (config) => {
-	const accessToken =
-		"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNzE2OTEwNTQzLCJpYXQiOjE3MTUxODI1NDMsImp0aSI6ImNlMDFhZTQ0MTM1YzQ4OTNiNGEyZGZlODQ4MjcxNzJlIiwidXNlcl9pZCI6MTl9.N9VLh0gnlSavHl-REGr7pWFvl3UEKkT-c24pj8QG_F8";
+	const userItem = localStorage.getItem("user");
+	const userData = userItem ? JSON.parse(userItem) : {};
+	const accessToken = userData.access || "";
```
Comment (assawalhy) [inline lib/axios.ts:11]: Why not cookies? How can we use these tokens in the server?

Comment (mohamed-ahmed-yousef) [reply lib/axios.ts:11]: - From next docs

image

Comment (mohamed-ahmed-yousef) [reply lib/axios.ts:11]: - I have an idea, we can use server action and client request to set `user` object in the `localStorage` and `session`  during `login` process and make a separate `http` for each one.
- but for now, we can skip it to save time

Comment (assawalhy) [reply lib/axios.ts:11]: How you get statistics in the server?

Comment (mohamed-ahmed-yousef) [reply lib/axios.ts:11]: I got it in the client

Comment (assawalhy) [reply lib/axios.ts:11]: Ok it is in the client. We can skip it for now, but it won't take time though.

**Review by assawalhy — APPROVED** [review]
Good

**Pull request merged**